Makefile

```diff
@@ -28,15 +28,15 @@ spec:
 		bundle exec rspec
 
 lint_yaml:
-	$(YAMLLINT) -c yamllint.yml .
+	find . -name '*.yml' | grep -v -e '^\./vendor' | xargs $(YAMLLINT) -c yamllint.yml
```

Comment:
Would `-not -path './vendor/*'` do the same thing here without the added complexity of regexes and pipes?